### PR TITLE
893 fix jinjia version

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Show and hide of source repo url based on the selection of open source [#869](https://github.com/rokwire/rokwire-building-blocks-api/issues/869)
 - Saving multiple capabilities error [#858](https://github.com/rokwire/rokwire-building-blocks-api/issues/858)
 - Warning in Logs Building Block. [#881](https://github.com/rokwire/rokwire-building-blocks-api/issues/881)
+- Fix jinjia version. [#893](https://github.com/rokwire/rokwire-building-blocks-api/issues/893)
```
Comment (sandeep-ps) [inline CHANGELOG.md:22]:
```suggestion
- Fix the Jinja version. [#893](https://github.com/rokwire/rokwire-building-blocks-api/issues/893)
```

**Comment by sandeep-ps** [comment]
Please merge the latest development changes and link the relevant issue to this PR. Thanks.

**Comment by minump** [comment]
isnt the same fix required for catalog, contributions, loggingservice as well?

**Comment by sandeep-ps** [comment]
This has been applied to Logs BB already.

@bingzhang, Could you please apply the same fix to `contributions`, `contributions/catalog`, `auth-middleware-test-svc`, and `authservice` modules too?

File: contributions/catalog/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,6 @@
 Flask==1.1.1
+itsdangerous==2.0.1
+Jinja2==3.0.3
 pymongo==3.7.2
```
Comment (minump) [inline contributions/catalog/requirements.txt:4]:
```suggestion
werkzeug==2.0.2
```
Add this requirement as well.

File: contributions/catalog/requirements.txt [hunk 3]
```diff
@@ -1,4 +1,6 @@
 Flask==1.1.1
```
Comment (minump) [inline contributions/catalog/requirements.txt:1]:
```suggestion
Flask==1.1.1
werkzeug==2.0.2
```
add additional requirement

**Review by ywkim312 — DISMISSED** [review]
Built docker and tested using the docker in both profiles and logging and all of them worked okay. Logging is not included in this one but worked okay.

File: contributions/catalog/requirements.txt (hunk 3 above)
Comment (sandeep-ps) [reply contributions/catalog/requirements.txt:1]: Hi, @bingzhang, it looks like this suggestion was resolved, but the `werkzeug` change has not been added to the catalog/requirements.txt. Please fix this. I'm unresolving this comment. Thanks.

Comment (sandeep-ps) [reply contributions/catalog/requirements.txt:1]: Thanks, Bing. I will mark this resolved.

**Comment by sandeep-ps** [comment]
@ywkim312, have you also tested Contributions BB?

**Comment by ywkim312** [comment]
> @ywkim312, have you also tested Contributions BB?

Didn't minu test?

**Comment by minump** [comment]
> > @ywkim312, have you also tested Contributions BB?
> 
> Didn't minu test?

Yup. I did,, contributions BB worked.

**Review by minump — APPROVED** [review]
approved

**Review by sandeep-ps — APPROVED** [review]

**Comment by sandeep-ps** [comment]
Thanks, All. I will merge this.

**Pull request merged**